.gitignore

```diff
@@ -14,6 +14,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
That's a good idea. Just try to remember: when using `.env*`, always check if it's already part of the `.gitignore` 😉 before push.